server/index.js

```diff
@@ -220,6 +220,54 @@ app.post('/api/sets', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/goals/:id', (req, res, next) => {
+  const userId = req.params.id;
+  const params = [userId];
+  const sql = `
+    select *
+      from "goals"
+     where "userId" = $1
+  `;
+  db.query(sql, params)
+    .then(result => {
+      res.status(200).json(result.rows);
+    })
+    .catch(err => next(err));
+});
+
+app.post('/api/goals/:id', (req, res, next) => {
```

Comment:
Now that you have log in/ log out, you should use `req.user` (if you have auth middleware) so that clients can't mess with data that they don't own. So all (almost all?) of your routes that just have the `userId` in the `req.params` need to be updated so that the `userId` comes from `req.user` instead.